packages/server/graphql/types/TeamPromptResponse.ts

```diff
@@ -0,0 +1,55 @@
+import {GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import Team from './Team'
+
+const TeamPromptResponse: GraphQLObjectType = new GraphQLObjectType<any, GQLContext>({
+  name: 'TeamPromptResponse',
+  description: 'A response of a single team member in a team prompt',
+  isTypeOf: ({status}) => !!status,
+  fields: () => ({
+    createdBy: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The userId that created the item'
+    },
+    createdByUser: {
+      type: new GraphQLNonNull(require('./User').default),
+      description: 'The user that created the item',
+      resolve: ({createdBy}, _args: unknown, {dataLoader}: GQLContext) => {
+        return dataLoader.get('users').load(createdBy)
+      }
+    },
+    content: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'the content of the response'
+    },
+    plaintextContent: {
```

Comment:
-2 `plain**T**extContent` is the column name in DB

Reply:
hmm, they actually match! see the discussion here: https://github.com/ParabolInc/parabol/pull/6068/files#r809458469

Reply:
Weird... On my local machine, it was capitalized T. But I checked prod it's lowercase so we are good.

![](https://user-images.githubusercontent.com/1879975/159669551-6b29393c-4ece-4011-a53f-96edb705194f.png)

Reply:
seems like an artifact of the old migration